### DISCUSSION TITLE
Bulk delete jobs

### DISCUSSION THREAD

**Discussion by TobiasKrok** [jhuckaby/Cronicle#807] [Q&A] [answered]
Hi!

Is there way to bulk delete jobs? I have a bunch that I don't need anymore and can't see any way to delete multiple jobs at once. 

I suppose otherwise I would need to export the DB and and remove all the jobs I don't need there and then import? That would at least be faster than clicking on every job and deleting. 

Thank you!

**Comment by jhuckaby** [comment, marked as answer]
Hi there!  I'm so sorry, but there isn't really a good way to do this with the current version of Cronicle.  Bulk search & delete is coming in v2, however, which will be released in early 2025.

For the current version of Cronicle (v0), your options are limited, but there are a few.  You will have to delete all the jobs -- there is no easy way to pick & choose.

The easiest way is what you described.  Perform a data export, delete ALL the data, then do an import.  The data export file doesn't contain jobs, so this will "work".  You do lose things like the admin activity log, however.

**Reply by TobiasKrok** [reply]
I see, that will be a nice improvement at least :) 

Thanks for your reply!
